index.js

```diff
@@ -121,4 +111,75 @@ app.post('/PostRestaurant', (request,response) =>{
 });
 
 
+
+
+
+
+
+
+app.get('/Searchpage',(request,response) =>{
+  pool.query('SELECT * FROM users',(error,results) =>{
+    if (error){
+      throw error;
+    } 
+    pool.query('SELECT * FROM restaurants',(error,results1) =>{
+      if (error){
+        throw error;
+      }
+      var result={'rows':results.rows,'rows2':results1.rows};
+      response.render('pages/Search-RestUser',result);
+    })
+   
+  })
+
+  app.post('/UsrSearch',(request,response) =>{
+    const {Svar}=request.body;
+   
+    pool.query('SELECT * FROM users WHERE firstName=$1 OR lastName=$1 OR city=$1',[Svar],(error,results) =>{
+      if (error){
+        throw error;
+      }
+    
+    pool.query('SELECT * FROM restaurants', (error,results2) =>{
+      if (error){
+        throw error;
+      }
+      var result={'rows':results.rows,'rows2':results2.rows};
+      response.render('pages/Search-RestUser',result)
+    })
+      
+    } )
+  })
+
+
+
+
+
+  app.post('/RestrSearch',(request,response) =>{
+    const {Svar}=request.body;
+   
+    pool.query('SELECT * FROM users',(error,results) =>{
+      if (error){
+        throw error;
+      }
+    
+    pool.query('SELECT * FROM restaurants WHERE name=$1 OR city=$1 OR address=$1',[Svar], (error,results2) =>{
```

Comment:
this query is not going to work because the user will have to add the exact address or name of the restaurant to find the restaurant, we need to do a LIKE comparison, also you need to look at the restaurant descriptions also